std_msgs/Header header
std_msgs/Float64 area
geometry_msgs/Point point1